tools/installpsh-suse.sh

```diff
@@ -154,10 +161,27 @@ if [[ ! -r "$package" ]]; then
     exit 1
 fi
 
-sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
-sudo zypper --non-interactive install "./$package"
+## Create the target folder where powershell will be placed
+$SUDO mkdir -p /opt/microsoft/powershell/$release
+## Expand powershell to the target folder
+$SUDO tar zxf $package -C /opt/microsoft/powershell/$release
+
+## Change the mode of 'pwsh' to 'rwxr-xr-x' to allow execution
+$SUDO chmod 755 /opt/microsoft/powershell/$release/pwsh
+## Create the symbolic link that points to powershell
+$SUDO ln -s /opt/microsoft/powershell/$release/pwsh $pwshlink
+
+## Add the symbolic link path to /etc/shells
+if [ ! -f /etc/shells ] ; then
+    echo $pwshlink | $SUDO tee /etc/shells ;
+else
+    grep -q "^${pwshlink}$" /etc/shells || echo $pwshlink | $SUDO tee --append /etc/shells > /dev/null ;
+fi
+
+## Remove the package
```

Comment:
Maybe better to change this comment to "Remove the downloaded package file"

Reply:
Sure, will fix it.